index.d.ts

```diff
@@ -180,6 +180,63 @@ export interface ConvertColor {
 	hexToAnsi(hex: string): number;
 }
 
+/**
+Basic modifiers.
+*/
+export type ModifierName =
```

Comment:
Just get the key from `Modifier` programmatically instead of defining these manually.